modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_address.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
     - node.type.branch
   module:
     - address
+    - datalayer
```

Comment:
can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done

Reply:
Actually datalayer added to dependencies here because it enabled to expose in third_party_settings below